src/lib.rs

```diff
@@ -129,6 +130,10 @@ async fn load_macaroon(path: impl AsRef<Path> + Into<PathBuf>) -> Result<String,
 ///
 /// If you have a motivating use case for use of direct data feel free to open an issue and
 /// explain.
+#[tracing::instrument(
+    name = "Connecting to LND",
+    skip(address, cert_file, macaroon_file),
```

Comment:
These shouldn't be sensitive if that's the reason why you skip them. (`macaroon_file` is just a path and on a correctly configured system it's only accessible by authorized users)

Reply:
I was actually skipping them because they did not implement debug as they were coming in, rather than them being sensitive. =)